### ISSUE TITLE
Edge: Problems dragging files from open dialog

### ISSUE THREAD

**Issue by kevinswartz** [react-dropzone/react-dropzone#958]
**Do you want to request a *feature* or report a *bug*?**

- [x] I found a bug
- [ ] I want to propose a feature

**What is the current behavior?**
I'm noticing an issue dragging files from an open file dialog onto the dropzone element in MS Edge only.  Sometimes the isDragActive and isDragAccept flags can get stuck to true.

**If the current behavior is a bug, please provide the steps to reproduce.**
If you drag a file over your dropzone, don't drop it, instead drag it back within your browse window and cancel the dialog, the isDragActive and isDragAccept flags can be improperly stuck to true.  You can see this in the "Using styled components" section on the dropzone demo site.

I also see this using the code below:
```
<Dropzone
            onDrop={this.onDrop}
            accept={fileTypes || null}
            multiple={true}
            noClick={true}
            noKeyboard={true}
          >
            {(params) => {
              let {getRootProps, getInputProps, isDragActive, isDragAccept, isDragReject} = params;
              return (
                (
                  <div {...getRootProps()}>
                    <input {...getInputProps()} />
                    { isDragActive &&  <div style={styles.overlayStyle}>Drop files here</div> }
                    {children}
                  </div>
                )
              )
            }}
          </Dropzone>
```

**What is the expected behavior?**
I would expect these flags to go back to false once the dialog is closed.

**Please mention other relevant information such as the browser version, Operating System and react-dropzone version.**
Tested on react-dropzone v10.2.1, and v10.2.2
I'm using React 16.5.2
Window 10 v1909 build 18363.720
MS Edge v44.18362.449.0

**Comment by rolandjitsu** [comment]
@kevinswartz I don't think that dragging files from an open file dialog is the intended way of using this lib. If you open the file selector, then just select the files you want and click open (or whatever the accept button label is).

The props getting stuck might be fixed by #938 . So give it a try and let us know if it's fixed. Otherwise I can only debug this in Edge for mac and it might behave differently.

**Comment by kevinswartz** [comment]
Thanks @rolandjitsu but it's not me wanting to do this, it's coming from a customer.  :)

I totally agree it's a weird case, but it does work in non-MS Edge browsers.  I tried in Edge for Mac, and it works just fine, probably because that's still built on Chromium.  

I looked through the changes in #938 but didn't see anything that would effect me here.  

In case it helps, I notice that my description of the isDragActive flag getting stuck isn't totally accurate.  I  added some logging as a child of my Dropzone render, and I notice that render gets called with isDragActive set to false, followed immediately by another render with it set to true.  I don't see a flicker or anything, I think the events come through together.
